### PR TITLE
Resolving CI failures due to latest numpy versions

### PR DESCRIPTION
**Pull request by Yash-10**
Resolving the `mypy` errors (#1100) due to the latest versions of numpy (`1.20.*`).

Thanks!!

### CODE REVIEW

File: src/poliastro/core/elements.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import numpy as np
 from numba import njit as jit, prange
 from numpy import cross
-from numpy.core.umath import cos, sin, sqrt
+from numpy.core.umath import cos, sin, sqrt  # type: ignore
```
Comment (astrojuanlu) [inline src/poliastro/core/elements.py:8]: Does

```
from numpy import cos, sin, sqrt
```

fix the problem?

**Review by astrojuanlu — COMMENTED** [review]
Thanks for this @Yash-10 ! Left only one comment

File: src/poliastro/core/elements.py (hunk 1 above)
Comment (Yash-10) [reply src/poliastro/core/elements.py]: Oh yes, thank you for the correction!

**Review by astrojuanlu — APPROVED** [review]

**Comment by astrojuanlu** [comment]
This addresses part of the CI breakage and the other part is on the works, thanks @Yash-10 !

**Pull request merged**